From 9299c274cb508686612181685b8cc828101c10e4 Mon Sep 17 00:00:00 2001
From: Xu Liang <liang.xu@intel.com>
Date: Mon, 15 Jun 2020 16:27:57 +0800
Subject: [PATCH] DRVLIB_SW-3662: GSW API - fix LMAC address. This is not a
 problem in kernel and the fix is for emulation module test when macro
 CHIPTEST is enabled.

---
 drivers/net/ethernet/lantiq/switch-api/mac/lmac_api.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/lantiq/switch-api/mac/lmac_api.h b/drivers/net/ethernet/lantiq/switch-api/mac/lmac_api.h
index 2448ac52aa6a..5837fb8c4193 100644
--- a/drivers/net/ethernet/lantiq/switch-api/mac/lmac_api.h
+++ b/drivers/net/ethernet/lantiq/switch-api/mac/lmac_api.h
@@ -13,8 +13,8 @@
 #include <xgmac_common.h>
 #if defined(PC_UTILITY) && PC_UTILITY
 #define LEGACY_MAC_BASE			0x42000
-#else
-#define LEGACY_MAC_BASE			0xb8420000
+#elif defined(CHIPTEST) && CHIPTEST
+#define LEGACY_MAC_BASE			0xb8442000
 #endif
 #define MAC_TEST			0x300
 #define MAC_PFADCFG			0x304
